feat: add fortune telling app (#62167)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-fortune-teller-app/68deccc163f7e0733208079d.md

---
id: 68deccc163f7e0733208079d
title: Step 6
challengeType: 0
dashedName: step-6
---

# --description--

> Now you have function that can help you to select element from the DOM, so for now your are going to create function that help to hide element from the DOM, declare `hideElements` function and pass `...elements` as argument and give it type of `HTMLElement[]`.

Reviewer: ```suggestion
Now you have a function that can select elements from the DOM. Next, create a function to hide elements from the DOM. Declare a `hideElements` function that takes `...elements` as a parameter with the type `HTMLElement[]`.
```

# --hints--

> You should have declared a function named hideElements.

Reviewer: ```suggestion
You should have declared a function named `hideElements`.
```
